godot-macros/src/class/derive_godot_class.rs

```diff
@@ -21,7 +21,7 @@ pub fn derive_godot_class(decl: Declaration) -> ParseResult<TokenStream> {
     let fields = parse_fields(class)?;
 
     let class_name = &class.name;
-    let class_name_str = class.name.to_string();
+    let class_name_str = struct_cfg.rename.unwrap_or(class.name.clone()).to_string();
```

Comment:
This always executes both `clone()` and `to_string()`.

`map_or_else()` or a good old `if let Some(...)` would be a bit nicer 🙂